app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -112,150 +112,164 @@
 
         <ScrollView
             android:layout_width="match_parent"
-            android:layout_height="0dp"
+            android:layout_height="match_parent"
             android:fillViewport="true"
-            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/notifications_container">
 
-            <androidx.constraintlayout.widget.ConstraintLayout
+            <RelativeLayout
```

Comment:
Nešlo by to udělat pouze přes jeden flat `ConstraintLayout` a groups?
Tyhle zanořené `RelativeLayouty` se mi vůbec v dnešné době už nelibí :(